### PR TITLE
Update patient ui schema

### LINKED ISSUE #1229: Rename schema_id to form_schema_id
As discussed here https://github.com/sussol/msupply/issues/12192

rename document::schema_id to form_schema_id

### LINKED ISSUE #1329: Programs > Patient Registration: move 'NUIC' field down
Would be more logical if the NUIC is at the  bottom of the `Patient Information` since you can only generate it if you have enter a value for each of its components. 

Screen Shot 2023-03-13 at 2 38 39 PM

### LINKED ISSUE #1331: Programs: rename a value for Marital Status field (patient registration form)
## What went wrong? 😲

Marital Status field in the patient registration/creation form. 

`Registered Partnership` should be renamed to `Cohabiting`

Screen Shot 2023-03-13 at 3 20 13 PM

### PR DESCRIPTION
**Pull request by clemens-msupply**
Fixes #1329
Fixes #1331 

# 👩🏻‍💻 What does this PR do? 
Smaller fixes to the patient ui schema.

### CODE REVIEW

**Comment by CarlosNZ** [comment]
> Fixes #1229

You mean #1329 right?

File: server/service/src/sync/program_schemas/patient_ui_schema.json [hunk 1]
```diff
@@ -150,10 +92,23 @@
...
+                        "Abau District",
+                        "Goilala District",
+                        "Kairuku-Hiri District",
+                        "Rigo District"
+                      ],
+                      "National Capital District": [
+                        "Moresby North-East",
+                        "Moresby North_West",
+                        "Moresby South"
+                      ],
```
Comment (CarlosNZ) [inline server/service/src/sync/program_schemas/patient_ui_schema.json:111]: Can we avoid these formatting changes if possible? (Mine reverts back to the original when I just save it)

File: server/service/src/sync/program_schemas/patient_ui_schema.json [hunk 2]
```diff
@@ -378,7 +411,7 @@
                       ["DIVORCED", "Divorced"],
                       ["WIDOWED", "Widowed"],
                       ["SEPARATED", "Separated"],
-                      ["REGISTERED_PARTNERSHIP", "Registered Partnership"]
+                      ["REGISTERED_PARTNERSHIP", "Cohabiting"]
```
Comment (CarlosNZ) [inline server/service/src/sync/program_schemas/patient_ui_schema.json:414]: You're just changing the UI? You don't think we need to update the enum values in the schema too? (Probably not, but just checking)

**Review by CarlosNZ — APPROVED** [review]
Cool, thanks. Couple of little comments, nothing important.

File: server/service/src/sync/program_schemas/patient_ui_schema.json (hunk 2 above)
Comment (clemens-msupply) [reply server/service/src/sync/program_schemas/patient_ui_schema.json:414]: The CAPITAL entries are the predefined values and the UI just displays what we need for this specific case. So we don't need to update the schema...

**Pull request merged**